language/fr/reg_mainpage.lang

```diff
@@ -0,0 +1,78 @@
+<?php
+
+if (!defined('INSIDE')) {
+    die('Accès refusé !');
+}
+
+include("{$_EnginePath}language/en/reg_mainpage.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Done.